metadata.json

```diff
@@ -12,6 +12,10 @@
     "foreman"
   ],
   "dependencies": [
+    {
+      "name": "camptocamp/systemd",
+      "version_requirement": ">= 2.2.0 < 3.0.0"
+    }<
```

Comment:
I believe you meant to make a comma here

Reply:
Updated